src/dataflowd/src/bin/dataflowd.rs

```diff
@@ -120,7 +120,7 @@ fn create_communication_config(args: &Args) -> Result<timely::CommunicationConfi
         let mut addresses = Vec::new();
         if args.hosts.is_empty() {
             for index in 0..processes {
-                addresses.push(format!("localhost:{}", 2101 + index));
+                addresses.push(format!("localhost:{}", 2102 + index));
```

Comment:
This `+ index` stuff will likely not work well with Linkerd policies.

Reply:
Yeah not to worry; this logic is for local development and only applies if you don't explicitly specify the host/ports to connect to.